etest/docker/container.py

```diff
@@ -73,9 +80,18 @@ def start(container: Container):
 
 def stop(container: Container, *args, **kwargs):
     """Stop a Docker container."""
-    return common.API_CLIENT.stop(container, *args, **kwargs)
+    return container.stop(*args, **kwargs)
 
 
 def wait(*args, **kwargs):
     """Wait for Docker container."""
     return common.API_CLIENT.wait(*args, **kwargs)
+
+
+def run(name: str, *args, **kwargs):
+    """Run a Docker container."""
+    logs = common.CLIENT.containers.run(name=name, **kwargs)
+    container = common.CLIENT.containers.get(name)
```

Comment:
This reads really funny to me.  Why do we run and then get the container?

Reply:
containers.run() doesnt return a container object. We have to get it manually.